### PR TITLE
Tests for some of the normative changes made to Temporal at the October TC39 plenary

### PR DESCRIPTION
**Pull request by ptomato**
Best examined commit by commit; each commit message has a link back to the normative PR into the Temporal proposal.

This is what I have so far, I will open subsequent test262 PRs for the others. Some of the changes are already covered by existing tests.

### CODE REVIEW

**Comment by ptomato** [comment]
I think that did it, thanks!

**Comment by rwaldron** [comment]
Pending response to https://github.com/tc39/proposal-temporal/pull/1862#issuecomment-956399464

**Comment by ptomato** [comment]
@rwaldron I replied there, that was indeed an oversight on my part.

**Comment by rwaldron** [comment]
@ptomato cool, thanks for the quick follow up :)

**Review by rwaldron — APPROVED** [review]

File: test/built-ins/Temporal/Duration/compare/relativeto-sub-minute-offset.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+---*/
+
+const duration1 = new Temporal.Duration(0, 0, 0, 31);
+const duration2 = new Temporal.Duration(0, 1);
+
+let relativeTo = "2000-01-01T00:00+00:45[+00:44:30.123456789]";
+assert.sameValue(Temporal.Duration.compare(duration1, duration2, { relativeTo }), 0, "rounded HH:MM is accepted in string");
+
+relativeTo = "2000-01-01T00:00+00:44:30[+00:44:30.123456789]";
+assert.throws(RangeError, () => Temporal.Duration.compare(duration1, duration2, { relativeTo }), "no other rounding is accepted for offset");
```
Comment (FrankYFTang) [inline test/built-ins/Temporal/Duration/compare/relativeto-sub-minute-offset.js:17]: I cannot see how this RangeError should be thrown.
the "+00:44:30.123456789" part in relativeTo is TimeZoneUTCOffsetName in the grammar, which is totally ignored by the spec text except syntax check .

**Pull request merged**